test/testingparams/default_nck.cfg

```diff
@@ -0,0 +1,10 @@
+[Preprocess]
+x_data_canc_files = [['cancer_gene_expression.tsv', ['Gene_Symbol']]]
+y_col_name = "colCONFIG"
+supplement = [('nck' , 'CONFIG')]
+input_dir = "/Users/koussanc/Desktop"
```

Comment:
We have to avoid absolute paths in the code and examples. This will break everytime someone else is executing it.

Reply:
This is the mock config I was using to play, it doesn't matter what it is.